src/optimization/Eliminator.h

```diff
@@ -106,6 +106,12 @@ namespace vc4c
 		 *   %x = add unif, %y
 		 */
 		void eliminateRedundantMoves(const Module& module, Method& method, const Configuration& config);
+
```

Comment:
Please add short documentation with some examples (you can take the ones from the source-file)